### PR TITLE
Use podman from debian bullseye

### PR DESCRIPTION
**Pull request by andrunko**
This removes podman-docker which is not available on debian bullseye.

https://phabricator.endlessm.com/T31300

### CODE REVIEW

**Comment by andrunko** [comment]
The CI failure seems to be unrelated to the changes here.

**Comment by wjt** [comment]
Does Debian provide a package with this alias? Or is it assumed that the user can do it?

**Review by wjt — APPROVED** [review]
LGTM but we can't merge this until the (indeed unrelated) build failure is resolved.

**Comment by wjt** [comment]
retest this please

**Comment by andrunko** [comment]
retest this please

**Comment by andrunko** [comment]
Landing after discussing with @dbnicholson to unblock other work since we need to land the change to drop `podman-docker`.

**Pull request merged**
